### PR TITLE
Remove Helm instructions from README

### LINKED ISSUE #120: Remove Helm instructions in README
We still need the Helm charts for the `uncharted` yaml, but we don't need to provide instructions for them on the README

### PR DESCRIPTION
**Pull request by trisberg**
Fixes #120

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ![](https://github.com/projectriff/charts/workflows/CI/badge.svg)
 
-# projectriff Helm charts (and uncharts)
+# projectriff Installation YAML
 
-Helm charts (and uncharts) to install Istio and riff.
+Installation YAML files for riff (includes Cert-Manager, Istio, Knative, Keda, KPack and Kafka dependencies).
```
Comment (scothis) [inline README.md]:
```suggestion
Release YAML files for riff (includes cert-manager, Istio, Knative, KEDA, kpack and Kafka dependencies).
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ![](https://github.com/projectriff/charts/workflows/CI/badge.svg)
 
-# projectriff Helm charts (and uncharts)
+# projectriff Installation YAML
```
Comment (scothis) [inline README.md]:
```suggestion
# projectriff Release YAML
```

File: README.md [hunk 3]
```diff
@@ -19,16 +19,27 @@ Helm charts (and uncharts) to install Istio and riff.
 
    ```sh
    riff_version=0.5.0-snapshot
+   ```
```
Comment (scothis) [inline README.md]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -19,16 +19,27 @@ Helm charts (and uncharts) to install Istio and riff.
...
 
+   ```sh
```
Comment (scothis) [inline README.md]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -190,7 +169,7 @@ Optionally, update the chart templates to the latest component builds.
 make templates
 ```
 
-Package charts locally. Charts will be placed in the `repository` directory.
+Package install YAML and charts locally. The install YAML will be placed in `uncharted` directory and charts will be placed in the `repository` directory.
```
Comment (scothis) [inline README.md]:
```suggestion
Package locally placing the YAML in the `uncharted` directory.
```

File: README.md (hunk 1 above)
Comment (trisberg) [reply README.md]: I have another issue for removing the riff charts #123

**Comment by trisberg** [comment]
We have another issue for removing the riff charts #123

**Pull request merged**